### PR TITLE
Leverage the GitHub's template chooser

### PR DESCRIPTION
**Pull request by phansys**
See "[Configuring the template chooser](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)".

### CODE REVIEW

File: project/.github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,52 +0,0 @@
-<!--
```
Comment (jordisala1991) [inline project/.github/ISSUE_TEMPLATE.md:1]: I think there is a specific way to tell dev-kit to remove a file, something related to name change with DELETE_ prefix. I do not remember exactly. Maybe @OskarStark can help here

Comment (phansys) [reply project/.github/ISSUE_TEMPLATE.md:1]: I guess you're right:
https://github.com/sonata-project/dev-kit/blob/22131a50897dbc52f357a38562d0a475901e09f2/src/Command/DispatchCommand.php#L493-L501

Comment (OskarStark) [reply project/.github/ISSUE_TEMPLATE.md]: Yes we need to prefix, deploy it and add another PR and delete the file from dev-kit

**Comment by OskarStark** [comment]
@greg0ire can you please deploy?

**Comment by greg0ire** [comment]
Done :)

**Comment by phansys** [comment]
If it was deployed yesterday, shouldn't we have some PRs open by dev-kit in their configured projects? :thinking:

**Comment by OskarStark** [comment]
Yes we should 👍🏻 Maybe it is not running due to the move projects dir PR... in this case we need to revert this one, merge the other PR, deploy and run and do this PR again...

**Comment by OskarStark** [comment]
Sorry I mean we need to revert the one which really deletes the files.

**Pull request merged**